gradle-jdks/build.gradle

```diff
@@ -66,4 +69,5 @@ tasks.withType(JavaCompile).configureEach {
 
 tasks.test {
     environment("PROJECT_VERSION", project.version)
+    dependsOn ':gradle-jdks-setup:fatJar'
```

Comment:
`GradleJdkInstallationSetupIntegrationTest` is inside the `integrationTest` source set, so I think this dependsOn needs to go on under `tasks.named('integrationTest').configure {`

Reply:
Hmm, that's actually in `gradle-jdks-setup`